### PR TITLE
Neckbeard

### PR DESCRIPTION
**Pull request by MM1953**

### CODE REVIEW

File: crawl-ref/source/dat/descript/backgrounds.txt [hunk 1]
```diff
@@ -173,6 +173,12 @@ Monk
 Monks start with a simple weapon of their choice, and gain additional piety
 with the first god they worship.
 %%%%
+Neckbeard
```
Comment (alexjurkiewicz) [inline crawl-ref/source/dat/descript/backgrounds.txt:181]: Needs to be just two lines (at 80char width)

**Review by alexjurkiewicz — CHANGES_REQUESTED** [review]
nice

**Comment by alexjurkiewicz** [comment]
I've merged a PR adding a bloatcrawl-specific README, please add the background description there as well

**Comment by MM1953** [comment]
I've added the new README.md with neckbeard's description to the neckbeard commit. I will be adding the neckbeard description to my quill orc PR to avoid merge conflicts, so make sure to merge this PR first if and when you merge quill orc.

File: crawl-ref/source/dat/descript/backgrounds.txt (hunk 1 above)
Comment (MM1953) [reply crawl-ref/source/dat/descript/backgrounds.txt:181]: Fixed

**Pull request merged**